src/test/java/com/automatedtest/sample/definitions/ScrollPageSteps.java

```diff
@@ -0,0 +1,36 @@
+package com.automatedtest.sample.definitions;
+
+import com.automatedtest.sample.pages.SettingPage;
+import com.automatedtest.sample.utils.Common;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.junit.Assert;
+
+public class ScrollPageSteps {
+    private final SettingPage settingPage;
+
+    public ScrollPageSteps() {
+        this.settingPage = new SettingPage();
+    }
+
+    @When("I Scroll to bottom")
+    public void aUserScrollToBottom() {
+        this.settingPage.scrollToBottom();
+    }
+
+    @And("I Scroll to top")
+    public void aUserScrollToTop() {
+        this.settingPage.scrollToTop();
+    }
+
+    @And("I capture image and save it to {string}")
+    public void aUserCaptureImage(String filePath) throws Exception {
+        this.settingPage.takeSnapShot(filePath);
```

Comment:
consider remove old image before taking a new one, so we can prevent verifying old pictures if something wrong happen with capture image

Reply:
Update In DriverUtils --> takeSnapShot